feature-libs/product-configurator/rulebased/core/state/reducers/configurator.reducer.ts

```diff
@@ -127,7 +127,9 @@ function handleActionUpdateConfigurationFinalizeSuccess(
   checkConflictSolverDialog(result);
   result.isCartEntryUpdateRequired = true;
   result.overview = undefined;
-  result.interactionState.newConfiguration = false;
+  if (state.interactionState.newConfiguration !== undefined) {
```

Comment:
If you set: 
`newConfiguration: source.newConfiguration ?? false`
then you could adjust this condition to
` if (state.interactionState.newConfiguration)`

Reply:
you have to see this condition together with whats happening in the create success action.
in the create success action we copy the newConfiguration state into the interaction state. Here we reset the newConfiguration state in the interaction state. This is required to force the popup in case we re-enter the configuration in the same session, as the configuration is then fetched from the local ngrx state instead o from the back-end.